trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfiguration.java

```diff
@@ -77,7 +77,13 @@ public boolean shouldOverridePins() {
     @Nullable
     public DomainPinningPolicy getPolicyForHostname(@NonNull String serverHostname) {
         // Check if the hostname seems valid
-        DomainValidator domainValidator = DomainValidator.getInstance(false);
+        DomainValidator domainValidator;
+        if (serverHostname.equals("127.0.0.1") || serverHostname.equals("localhost")) {
+            domainValidator = DomainValidator.getInstance(true);
+        } else {
+            domainValidator = DomainValidator.getInstance(false);
+        }
```

Comment:
Same question as above.

Reply:
@nabla-c0d3 
https://github.com/datatheorem/TrustKit-Android/blob/3f96ec34d5a1fd8a3bcf24bb7c572dc4fe48170a/trustkit/src/main/java/com/datatheorem/android/trustkit/config/DomainValidator.java#L155

Please note that the `allowLocal` flag should be true and it's false without this change...